installation: adjust make option (#626)
containers/website-new: docs/installation.md

Reviewer: You need EXTRA_BUILDTAGS here too
```suggestion
make EXTRA_BUILDTAGS="btrfs_noversion exclude_graphdriver_btrfs"
```
Please search for other instances.

```bash
git clone https://github.com/opencontainers/runc.git $GOPATH/src/github.com/opencontainers/runc
cd $GOPATH/src/github.com/opencontainers/runc

> make EXTRA_BUILDTAGS="selinux seccomp"

Reviewer: ```suggestion
make RUNC_BUILDTAGS="selinux"
```
runc default already includes seccomp

```bash
git clone https://github.com/containers/podman/
cd podman
make EXTRA_BUILDTAGS="selinux seccomp" PREFIX=/usr
sudo env PATH=$PATH make install PREFIX=/usr
```

#### Build Tags

If you want to build Podman and auto-detect tags, omit the `BUILDTAGS` option when running make.

```bash
make
sudo make install
```

Podman supports optional build tags for compiling support of various features.

> To add build tags to the make option the `BUILDTAGS` variable must be set, for example:

Reviewer: I think you want
```suggestion
To add build tags while keeping auto-detected defaults, set EXTRA_BUILDTAGS, for example:
```
